src/components/confirm-leave-mixin.cjsx

```diff
@@ -0,0 +1,12 @@
+$ = require 'jquery'
+
+module.exports =
+  # getFlux: -> {store, actions}
+  # getId: -> id 
+  componentDidMount: ->
+    $(window).on("beforeunload", @confirmLeave)
```

Comment:
Should we avoid jQuery?  I told Fred the other day to go ahead and use it where it was convenient.